### PR TITLE
Replace sanitize.css with codify.css

### PR DESCRIPTION
**Pull request by zdroid**
There has been a name dispute between my project and the other sanitize.css. I've decided to completely rebrand my project to codify.css and start from v1.0.0 so there are no such issues in future. Previous PR got ignored by bot.

I hope you understand the decision and implement this change. Thanks.

### CODE REVIEW

**Comment by zdroid** [comment]
@jsdelivrbot Please re-check

**Comment by zdroid** [comment]
@jimaek Can you make the bot re-validate this?

File: files/codify.css/update.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "packageManager": "bower",
+  "name": "codify.css",
+  "repo": "zdroid/codify.css"
```
Comment (megawac) [inline files/codify.css/update.json]: `"files": {}` is required

Comment (zdroid) [inline files/codify.css/update.json]: I think I've fixed that now.

**Pull request merged**